tokio/src/net/windows/named_pipe.rs

```diff
@@ -2305,11 +2312,11 @@ impl ClientOptions {
     /// automatically when using this method.
     ///
     /// [`CreateFile`]: https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilea
-    /// [`SECURITY_IDENTIFICATION`]: crate::winapi::um::winbase::SECURITY_IDENTIFICATION
+    /// [`SECURITY_IDENTIFICATION`]: windows_sys::Win32::Storage::FileSystem::SECURITY_IDENTIFICATION
```

Comment:
As far as I can tell, this link doesn't actually work. It just goes to `tokio/net/windows/named_pipe/windows_sys::Win32::Storage::FileSystem::SECURITY_IDENTIFICATION`

I'm not sure why.

Reply:
Yes, just noticed that on the netlify host as well. I guess we can go back to the links. 🤷 Not sure what else to try.

Reply:
Oh, it looks like the doc examples need it to be a `dev-dependencies` dependency but the links need a `dependencies` dependency. 😳

Perhaps something like this:

```
[target.'cfg(windows)'.dependencies.windows-sys]
version = "0.42.0"
optional = true

[target.'cfg(docsrs)'.dependencies.windows-sys]
version = "0.42.0"
features = [
    "Win32_Foundation",
    "Win32_Security_Authorization",
]

[target.'cfg(docsrs)'.dev-dependencies.windows-sys]
version = "0.42.0"
features = [
    "Win32_Foundation",
    "Win32_Security_Authorization",
]
```

Reply:
Or perhaps the simpler form would work:

```
[target.'cfg(windows)'.dependencies.windows-sys]
version = "0.42.0"
optional = true

[target.'cfg(docsrs)'.dependencies.windows-sys]
version = "0.42.0"
features = [
    "Win32_Foundation",
    "Win32_Security_Authorization",
]
```

Sorry I can't confirm this locally as this doesn't seem to build at all on Windows.

Reply:
The latter seems to work in a minimal repro. I'll do a bit more testing and then push that and we'll see.

Reply:
If you can find something that works, then that's great. Otherwise its fine to go back to the links. Let me know when you want me to look again.

Reply:
Unfortunately, that didn't work for some reason. I've switched back to using links.